_pages/dev/extending-checkout.md

```diff
@@ -394,6 +394,55 @@ provideConfig({
 
 **Note:** You can use this same approach to combine all the steps into a single-step checkout.
 
+## Multiple Checkout Flows
+
+Multiple checkout flows allows you to provide realtime switching between checkout flows, and eliminates the need to rebuild your storefront app for each change. This allows you to provide multiple checkout flows to cater to different market segments or product types.
+
+**Note:** Multiple checkout flows requires Spartacus 2211.x
+
+### Configuring Multiple Checkout Flows
+
+The `flows` checkout out property allows you to inject different checkout flows using specific keys. When the checkout process is initiated, the checkout orchestrator service queries the back end for the `paymentProvider` value. You can set a separate `paymentProvider` value for each base store. If the returned `paymentProvider` value matches a predefined checkout flow key, that specific flow is served to the customer. For example, if the back end returns a value of `ProviderA` for the `paymentProvider`, and there is a corresponding checkout flow with the `ProviderA` key, then the `ProviderA` checkout flow is activated.
+
+**Note:** Using multiple checkout flows in optional. If a specific checkout flow key is not found, the service falls back to the standard checkout steps and configuration.
+
+The following is an example of a `checkout.flows` configuration:
+
```

Comment:
I'm not sure if `checkout.flows` is the correct annotation for referring to `checkout : { flows: { ... `
If not, I can probably just change it to "`flows` configuration".

Reply:
In theory it's correct, as it refers to object hierarchy. But, I would prefer just to use "`flows` configuration" in this case.

Reply:
Perfect! 🙂